cedar-drt/src/lean_impl.rs

```diff
@@ -52,40 +53,40 @@ extern "C" {
     fn initialize_DiffTest_Main(builtin: i8, ob: *mut lean_object) -> *mut lean_object;
 }
 
-#[derive(Debug, Deserialize)]
-struct ListDef<String> {
-    l: Vec<String>,
-}
+pub const LEAN_AUTH_MSG: &str = "Lean authorization time (ns) : ";
```

Comment:
[nit] personally I'd prefer milliseconds.

Reply:
I don't feel strongly either way, but I like consistency, and the rest of the code uses ns. Do you think it's worth changing the Rust & Dafny timings to use ms too?